packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -2641,3 +2690,149 @@ Policy Rule conditions aren't supported for this policy.
+
+## Entity risk policy
+
+<ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />
+
+The Entity risk policy specifies what action or task to execute in reaction to a risk event. The type is specified as `ENTITY_RISK`.
+
+#### Entity risk policy example
+
+```json
+    {
+        "type": "ENTITY_RISK",
+        "name": "useful example name here",
+        "description": "useful example description here"
+    }
+```
+
+### Policy conditions
+
+Policy conditions aren't supported for this policy.
+
+### Policy Rules conditions
+
+You can apply the following conditions to the rules associated with an Entity risk policy:
+
+* [People condition](#people-condition-object)
+
+* [Risk score condition](#entity-risk-score-condition-object)
+
+* [Risk detection condition](#entity-risk-detection-condition-object)
+
+### Entity risk object
+
+The Entity risk object indicates the next steps to take in response to a risk event. The object is specified as `entityRisk`.
+
+| Property                | Description              | Data Type                                       | Required                      | Default |
+| ---                     | ---------------          | ---                                             | ---                           | ---     |
+| `actions`               | The action to take based on the risk event.              | Array of [action value objects](#actions-array-object-values)                         | Yes                           | `[]`   |
+
+#### Actions array object values
+
+The `entityRisk` object's `actions` array can be empty or contain one of two `action` object value pairs. This object determines the specific response to a risk event.
+
+| Array value               | Description              | Data Type                                       | Required                     | Default |
+| ---                     | ---------------          | ---                                             | ---                           | ---     |
+| `[]`                 | This action only logs the user risk event.             | object                      |  Yes                      | Yes  |
+| `[ { "action": "TERMINATE_ALL_SESSIONS" } ]`              | This action revokes or terminates all of the user's active sessions.             | object                      |       No   | No
+| `[ { "action": "RUN_WORKFLOW", "workflow": {"id": "123123123"} } ]`               | This action runs a workflow and must include the additional workflow `id` for the `workflow` property.            | object                      | No                       | No   |
+
+#### Entity risk actions default example
+
+```json
+"actions": {
+  "entityRisk": {
+    "actions": []
+  }
+}
+```
+
+#### Entity risk actions workflow example
+
+```json
+"actions": {
+  "entityRisk": {
+    "actions": [ { "action": "RUN_WORKFLOW", "workflow": {"id": "123123123"} } ]
+  }
+}
+```
+
+## Continuous Access Evaluation (CAE) policy
```

Comment:
Can you follow up with @barbaravo-okta? It's my understanding that there isn't a Continuous Access Evaluation policy on its own.

Also, CAE acronym isn't in use

Reply:
Will remove the CAE reference, thanks. From an API perspective, it's a `policy_type` so have documented it as such, but will work with Barbara on this.